pom.xml

```diff
@@ -192,6 +192,12 @@
     </build>
 
     <dependencies>
+	    <dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.22</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
Why?????

I don't even see Lombok being used anywhere in your PR

Reply:
I'm using it inside MaterialConverter. For the @Null and @NonNull annotations.

Reply:
Nevermind, got rid of this.